syncpack.config.cjs

```diff
@@ -173,11 +173,15 @@ module.exports = {
 		{
 			label: "Versions of common Fluid packages should all match",
 			dependencies: [
+				"@fluid-internal/eslint-plugin-fluid",
+				"@fluid-tools/benchmark",
```

Comment:
This addition is taking out PR runs.
core-utils is on version 0.47.
Curious how this got through PR.

Reply:
Opened a fix PR: https://github.com/microsoft/FluidFramework/pull/22132. I'm looking into why this passed CI.

Reply:
Found the issue. Client pipeline was not triggered by edits to the syncpack config. The only job that _always_ runs for all PRs is the "Repo policy check", but it doesn't run any release-group-specific jobs. I'll open a second PR with a fix to trigger client when syncpack config is edited.

Reply:
Second PR: #22133